.travis.yml

```diff
@@ -6,5 +6,11 @@ python:
  - '3.7'
 install:
  - pip install flake8
+before_script: |
+ mv .travis/system.dic.test tests/resources/system.dic && \
+ wget https://object-storage.tyo2.conoha.io/v1/nc_2520839e1f9641b08211a5c85243124a/sudachi/sudachi-dictionary-20190531-core.zip && \
```

Comment:
The system dictionaries are updated about once a month. I don't want to rewrite these lines each time :)

Reply:
- removed these lines
- put-off test_tokenizer.py, test_dictionary.py (1) 

(1): I put-off to apply these tests on CI process. Because SudachiPy doesn't support `user.dic` but Java-ver tests use `user.dic`.